Source/WebCore/loader/ImageLoader.cpp

```diff
@@ -245,7 +245,7 @@ void ImageLoader::updateFromElement(RelevantMutation relevantMutation)
             Ref cachedResourceLoader = document->cachedResourceLoader();
             bool autoLoadOtherImages = cachedResourceLoader->autoLoadImages();
             cachedResourceLoader->setAutoLoadImages(false);
-            RefPtr page = m_element->document().page();
+            RefPtr page = protectedDocument()->page();
```

Comment:
Again, page() is a trivial getter so we shouldn't need to use `protectedDocument`.

Reply:
Fixed.